test/git-as.bats

```diff
@@ -486,6 +486,7 @@ load test_helper
 }
 
 @test "as duet: does not write Co-authored-by trailer when rebasing if GIT_DUET_CO_AUTHORED_BY" {
+  git config rebase.backend apply
```

Comment:
Can you please update this so this does not modify someone's local config if they run it on localhost? In the setup steps, it should save their previous setting (or the absence of this setting), and then restore that value (or delete the config) in the teardown step.

Reply:
Updated!